packages/fast-components-react-base/src/tabs/tab.tsx

```diff
@@ -9,10 +9,15 @@ class Tab extends Foundation<
     ITabUnhandledProps,
     {}
 > {
+    public static defaultProps: Partial<ITabHandledProps> = {
```

Comment:
`defaultProps` should be `Partial<TabProps>` - defaultProps are always a partial of the comprehensive prop object.